config/samples/ocm-api-serviceaccount.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+data:
```

Comment:
Probably want to show this as `stringData` too

Reply:
Updated the field to use `stringData`.